_delphi_utils_python/delphi_utils/geomap.py

```diff
@@ -399,10 +401,13 @@ def replace_geocode(
         date_col: str or None, default "date"
             Specify which column contains the date values. Used for value aggregation.
             If None, then the aggregation is done only on geo_id.
+        pop_col: str or None
+            Specify which columns contains population values. Used for correcting for double
+            counting if a megafips is present.
```

Comment:
```suggestion
            Specify which columns contains population values. Used to avoid double
            counting if a megafips is present.
```